src/store/migrations.js

```diff
@@ -210,17 +210,38 @@ const migrations = [
         }
       }
     }
+  },
+  { // rskLegacyDerivation
+    version: 11,
+    migrate: async (state) => {
+      const accKeys = Object.keys(state.accounts);
+
+      if (!accKeys.length) {
+        return {
+          ...state,
+        }
+      }
+
+      const accData = state.accounts[accKeys[0]].mainnet;
```

Comment:
`accounts` is indexed by wallet id. I think you need to iterate over the walletIds and override rsk for each of them. See other migrations how to do this